README.md

```diff
@@ -131,6 +133,8 @@ $ make stop language=node
   This repo should generally work with active LTS versions of each language such as node >= 12,
   python >= 3.8, ruby >= 2.6, etc.
 - Your environment variables populated in `.env`
+- [npm](https://www.npmjs.com/get-npm)
+- If using Windows, a command line utility (such as Powershell or Cygwin) capable of running basic Unix shell commands
```

Comment:
i dont believe powershell can run unix commands? you would need Cygwin or MinGW or gitbash or WSL etc.

Reply:
powershell doesn't have full Unix shell support, but it does have built-in aliasing for simple commands (like ls, touch, etc.) which I think is good enough for running the basic scripts we have